src/App.tsx

```diff
@@ -2,13 +2,27 @@ import React from 'react';
 
 import './App.scss';
 
-// import posts from './api/posts';
-// import comments from './api/comments';
-// import users from './api/users';
+import posts from './api/posts';
+import comments from './api/comments';
+import users from './api/users';
+import { PostList } from './components/PostList';
+
+const preparedPosts = posts.map(key => {
+  const user = users.find(tmpUser => tmpUser.id === key.userId) || null;
+  const postComments = comments.filter(tmpComment => tmpComment.postId === key.id) || null;
```

Comment:
```suggestion
const preparedPosts = posts.map(post => {
  const user = users.find(tmpUser => tmpUser.id === post.userId) || null;
  const postComments = comments.filter(tmpComment => tmpComment.postId === post.id);
```
Зачем после filter ставить null? если комментариев нет - результатом будет пустой массив

Reply:
точно)